rust/stackable-cockpitd/src/main.rs

```diff
@@ -23,6 +25,18 @@ mod cli;
 async fn main() -> Result<(), Whatever> {
     let cli = Cli::parse();
 
+    // Construct the tracing subscriber
+    let format = fmt::format()
+        .with_level(false)
+        .with_ansi(true)
+        .without_time();
```

Comment:
Why remove the level/time?

Reply:
CC @Techassi

Reply:
I added level and timestamp to the log, does that work for you @Techassi?

Reply:
Yes. Makes sense. I removed it for the `ctl`, as both the level and time are not super important in that context. But for `cockpitd`, this is totally fine.